integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDomainInPV.java

```diff
@@ -324,6 +332,49 @@ public void testIstioDomainHomeInPv() {
     boolean checkApp = checkAppUsingHostHeader(url, domainNamespace + ".org");
     assertTrue(checkApp, "Failed to access WebLogic application");
 
+    // Refer JIRA https://jira.oraclecorp.com/jira/browse/OWLS-86407
```

Comment:
update the use case description for this change

Reply:
Added the following line to the test description 

Additionally, the test verifies that WebLogic cluster can be scaled down and scaled up in the absence of Administration server.